### PR TITLE
fixed test cases issue and added csv option in target deserializer

### PR DESCRIPTION
**Pull request by badrinathpatchikolla**

### CODE REVIEW

File: src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala [hunk 1]
```diff
@@ -251,57 +251,6 @@ class Test extends FunSuite with BeforeAndAfter {
     }
   }
 
-  def testSourceSql(tableName: String): DataFrame = {
```
Comment (mantovani) [inline src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala:254]: Why did you delete those test cases ?

**Review by mantovani — CHANGES_REQUESTED** [review]

File: src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala (hunk 1 above)
Comment (badrinathpatchikolla) [reply src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala:254]: In previous PR these def got repeated, removed duplicated ones and tested

**Review by mantovani — APPROVED** [review]
